### ISSUE TITLE
Tabbed reports accessibility - tab icons require tooltips and aria-expanded

### ISSUE THREAD

**Issue by phudson-he** [archesproject/arches#12827]
**Tooltip Issue (1.1.1 Non-Text Content):**

1. Given I am a user with disabilities
2. When I hover or focus on the tab icons
3. Then a textual tooltip should appear with the name of the tab, informing the user what the tab is called before they click on it.

**Icon Expanded Issue (4.1.2 Name, Role, Value)**

1. Given I am a user reliant on the the screen reader
2. When I select any of the tab icons to expand the tab
3. Then the screen reader should inform the user of the title of the tab, and that the tab has been expanded.

**Expected Result:**
The user should be informed via the screen reader every time hidden content is expanded or collapsed, and should also be informed of the name of the content they are hiding/showing via aria labels.

**Actual Result:**
The icons are interpreted correctly by the screen reader in their collapsed state. However, when the user selects any of the icons to expand the tab, the screen reader does not inform the user that the tab they have selected has been expanded. Rather, the screen reader simply reads out all the body text contained within the tab.
